.github/templates/appengine-flexible-deploy-gcloud.mdpp

```diff
@@ -0,0 +1,21 @@
+## Deploying to App Engine
+
+To run the application locally, use the Maven Jetty plugin.
+
+    mvn clean jetty:run
+
+View the app at [localhost:8080](http://localhost:8080).
+
+To deploy the app to App Engine, run
+
+    mvn clean gcloud:deploy
```

Comment:
Removed this file. We should update our samples to use the new plugins soon, then.